concepticondata/conceptlists/Marchese-1983-311.tsv

```diff
@@ -0,0 +1,312 @@
+Marchese-1983-311-154	154	champ	212	FIELD	field	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-155	155	chapeau	771	HAT	hat	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-156	156	chose	738	THING	thing	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-157	157	concession			concession	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-158	158	conte	2937	FAIRY TALE	tale	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-159	159	demain	1329	TOMORROW	tomorrow	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-160	160	Dieu	1944	GOD	God	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-161	161	eau	948	WATER	water	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-162	162	étoile	1430	STAR	star	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-163	163	excrément	675	FAECES (EXCREMENT)	excrement	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-164	164	faim	1864	HUNGER	hunger	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-165	165	fer	621	IRON	iron	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-166	166	feuille	628	LEAF	leaf	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-167	167	funérailles	3336	FUNERAL	funeral	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-168	168	fumée	778	SMOKE (EXHAUST)	smoke	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-169	169	gale	3172	SCABIES	scabies	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-170	170	guerre	935	WAR	war	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-171	171	héritage			heritage	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-172	172	hier	1174	YESTERDAY	yesterday	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-173	173	homonyme			namesake	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-174	174	honte	1776	SHAME	shame	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-175	175	jour	1225	DAY (NOT NIGHT)	day	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-176	176	là-bas	3411	OVER THERE	over there	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-177	177	langue (parole)	1307	LANGUAGE	language (speech)	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-178	178	lune	1313	MOON	moon	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-179	179	maison	1252	HOUSE	house	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-180	180	matin	1339	MORNING	morning	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-181	181	médicament	1372	MEDICINE	drug	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-182	182	mensonge	2826	LIE (FALSEHOOD)	lie (n)	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-183	183	mer	1474	SEA	sea	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-184	184	mort	910	DEATH	death	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-185	185	natte	195	MAT	mat	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-186	186	nom	1405	NAME	last name	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-187	187	nouvelle	2964	NEWS	news	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-188	188	nourriture	256	FOOD	food	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-189	189	nuit	1233	NIGHT	night	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-190	190	odeur	3704	ODOR (SMELL)	odor	LES SUBSTANTIFS GENERAUX
+Marchese-1983-311-191	191	oeuf	744	EGG	egg	LES SUBSTANTIFS GENERAUX
```

Comment:
same here, not "o e u f", but rather "œ u f"

Reply:
No, the spelling here is exactly like in the source.

Reply:
Then forget about my comment.